ci/dockerfiles/ansible-e2e-hybrid.Dockerfile

```diff
@@ -23,16 +23,21 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 # Ensure fresh metadata rather than cached metadata in the base by running
 # yum clean all && rm -rf /var/yum/cache/* first
 RUN yum clean all && rm -rf /var/cache/yum/* \
- && (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm || true) \
- && yum install -y python-devel gcc inotify-tools \
- && easy_install pip \
- && pip install -U --no-cache-dir setuptools pip \
- && pip install --no-cache-dir --ignore-installed ipaddress \
-      ansible-runner==1.2 \
-      ansible-runner-http==1.0.0 \
-      openshift==0.8.9 \
-      ansible==2.8 \
- && yum remove -y gcc python-devel \
+ && (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm || true) \
+ && yum install -y python36-devel.x86_64 gcc \
+ # Install inotify-tools
+ && yum install -y epel-release \
```

Comment:
epel-release-latest-7.noarch -> epel-release-latest-8.noarch.rpm
python-devel -> python36-devel.x86_64
inotify-tools -> replaced from download and install of itss rpm pkg since it has been not came so longer added in the rhel8.